src/lib/api/inline.preboot.code.ts

```diff
@@ -111,9 +115,7 @@ export function getEventRecorderCode(): string {
  * @param customOptions PrebootRecordOptions that override the defaults
  * @returns Generated inline preboot code is returned
  */
-export function getInlinePrebootCode(customOptions?: PrebootOptions): string {
-  const opts = <PrebootOptions>assign({}, defaultOptions, customOptions);
```

Comment:
I don't want to move this because the Module is not the only consumer for this (ie React users may call it manually, and still want the default options)

Reply:
I see. Should I extend in both places or add an undefined check?

My concern with checking for undefined is that it breaks if the default changes.

Reply:
When would the default change?

Reply:
Maybe it's not a big problem... Is extending in both places OK?

Reply:
Why would we need it in both places if the `inlinePrebootCode` handles it anyway?